cap/modules/deposit/cli.py

```diff
@@ -87,3 +88,79 @@ def add(file_path, schema, version, egroup, usermail, limit):
                             fg='green')
 
         db.session.commit()
+
+
+@fixtures.command('create-record')
+@click.option('--data', '-d',
+              type=click.Path(exists=True),
+              help='JSON data file')
+@click.option('--ana', '-a',
+              help='Type of analysis',)
+@click.option('--role', '-r',
+              help='Role with access to the record')
+@click.option('--user', '-u',
+              help='User with access to the record')
+@click.option('--owner', '-o',
+              help='Owner of the record')
+@with_appcontext
+def create_record(data, ana, role, user, owner):
+    """
+    Create a new deposit through the CLI.
+    Usage:
+        cap record create --data DATAFILE --role ROLE --user USER --owner OWNER
+    """
+    try:
+        with open(data) as datafile:
+            data = json.load(datafile)
+    except (KeyError, ValueError):
+        raise click.BadParameter('Not a valid JSON file.')
+
+    if isinstance(data, dict):
+        update_data_with_schema(data, ana)
+        create_deposit_with_permissions(data, user, role, owner)
+    else:
```

Comment:
Will never go here for an array. Right?

Reply:
it will, the else handles that. however, i will improve it because "hello" is also a valid json, so we can make it a bit more explicit